src/StoreApi/Schemas/V1/AbstractAddressSchema.php

```diff
@@ -90,18 +93,24 @@ public function get_properties() {
 	 */
 	public function sanitize_callback( $address, $request, $param ) {
```

Comment:
We sanitize all incoming fields, include additional ones. Currently they're treated as text, in the future we would include things like select and checkboxes.